share/gpodder/ui/gtk/gpodder.ui

```diff
@@ -149,16 +149,6 @@
             <property name="hexpand">True</property>
             <property name="vexpand">True</property>
             <child>
-              <object class="GtkNotebook" id="wNotebook">
-                <property name="visible">True</property>
-                <property name="can_focus">True</property>
-                <property name="show_tabs">True</property>
-                <property name="show_border">True</property>
-                <property name="tab_pos">GTK_POS_TOP</property>
-                <property name="scrollable">False</property>
-                <property name="enable_popup">False</property>
-                <signal handler="on_wNotebook_switch_page" name="switch_page"/>
-                <child>
```

Comment:
I intentionally didn't change the indentation on any existing lines to not bloat the diff and keep it easy to rebase. I can re-indent everything before this gets merged.